sui_programmability/examples/games/sources/drand_lib.move

```diff
@@ -40,6 +40,8 @@ module games::drand_lib {
         // Convert round to a byte array in big-endian order.
         let round_bytes: vector<u8> = vector[0, 0, 0, 0, 0, 0, 0, 0];
         let i = 7;
+
+        // Note that this loop never copies the last byte of round_bytes, though it is not expected to ever be non-zero.
```

Comment:
Was this comment supposed to be in a different PR?

Reply:
just noticed it here and added to clarify